### PR TITLE
Update to account for the latest theme changes

### PR DESCRIPTION
**Pull request by rebloor**
A checklist to help your pull request get merged faster:
- [ ] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [ ] Link to related issues or pull requests, if any

### CODE REVIEW

**Comment by vinyldarkscratch** [comment]
Hey @rebloor -- thanks for your PR, however there's a couple of issues before we can properly review this:

1) It looks like some of the "status" blocks were moved above the "support" blocks.  Can you please move these back?
2) It seems you have single-entry arrays within your data, making the array part redundant.  Can you turn that into just the dictionary and scrap the array?  This will satisfy the linter.
3) Most importantly, can you provide background to your changes?  We'll be able to review your PR much easier if we know the context to these changes!

(Note: to check to see if your changes satisfy the linter, you can run `npm test`.  :wink:)

File: webextensions/manifest/theme.json [hunk 1]
```diff
@@ -45,6 +45,11 @@
           },
           "accentcolor": {
             "__compat": {
+              "status": {
```
Comment (vinyldarkscratch) [inline webextensions/manifest/theme.json]: These `status` fields should actually be below the `support` block (yes, I know it's not alphabetical sorting).  Can you swap the location of the `status` block to go right after the `support` one as to maintain consistency with the rest of the files in BCD?

**Comment by vinyldarkscratch** [comment]
Thanks for the updates, as well as the [link to the Bugzilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1472740#c34)!  That's all from my end -- @irenesmith, take it away.  :wink:

**Comment by irenesmith** [comment]
> 
> 
> Thanks for the updates, as well as the [link to the Bugzilla bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1472740#c34)! That's all from my end -- @irenesmith, take it away. 😉

Could you approve the changes, please? I'd like to merge this PR

**Pull request merged**